### PR TITLE
Database refactor

### PR DESCRIPTION
**Pull request by dominicfraser**
The database tables have been completely refactored and updated to reflect V1 plans, excluding the join tables, still to be added.

The player.wins() logic has been updated and tested with the current landing page.

THIS MERGE WILL BREAK adding a new player and adding a new game - but is needed in order for those responsible for those pages to have the new db and models to begin to update those views.  Don't merge into master until these are working again.

Refactoring db will continue by adding the join tables.

### CODE REVIEW

File: models/game.rb [hunk 1]
```diff
@@ -2,22 +2,30 @@
...
 
   ### INSTANCE METHODS
 
-def save()
-  sql = "INSERT INTO games (p1_id, p2_id, winner) VALUES (#{@p1_id}, #{@p2_id}, #{@winner}) RETURNING id"
-  game_array_pg = SqlRunner.run(sql)
-  @id = game_array_pg.first['id'].to_i
-end
+  def save()
+    sql = "INSERT INTO games (p1_id, p2_id, p1_score, p2_score, p1_org_id, p2_org_id, p1_group_id, p2_group_id, location_id) VALUES (#{@p1_id}, #{@p2_id}, #{@p1_score}, #{@p2_score}, #{@p1_org_id}, #{@p2_org_id}, #{@p1_group_id}, #{@p2_group_id}, #{@location_id}) RETURNING *"
```
Comment (DewarM) [inline models/game.rb]: nit: I think this query should be split over multiple lines to make more readable: 
"INSERT INTO games (
p1_id,
p2_id...
"

File: models/player.rb [hunk 2]
```diff
@@ -2,23 +2,34 @@
...
+    @picture = player_hash['picture']
+    @primary_org_id = player_hash['primary_org_id'].to_i
+    @primary_group_id = player_hash['primary_group_id'].to_i
   end
 
   ### INSTANCE METHODS
 
   def save()
-    sql = "INSERT INTO players (p_name) VALUES ('#{@p_name}') RETURNING id"
+    sql = "INSERT INTO players (p_name, rating, picture, primary_org_id, primary_group_id) VALUES ('#{@p_name}', #{@rating}, '#{@picture}', #{@primary_org_id}, #{@primary_group_id}) RETURNING id"
```
Comment (DewarM) [inline models/player.rb]: nit: also split over multiple lines

**Review by DewarM — CHANGES_REQUESTED** [review]
Do we want to commit these changes even though it breaks develop? Should that be another branch or should we wait until the refactor is completed? I'm not sure.

**Comment by dominicfraser** [comment]
@DewarM broke sql over multiples lines.
I'm also not sure - as its just development I say merge it - as we know we are changing the ADD pages, and they can't be correctly changed without this information to work from?  Whereas we wouldn't merge to master, as that would mean our 'live' product would be broken.

**Review by DewarM — APPROVED** [review]

**Pull request merged**